CMakeLists.txt

```diff
@@ -371,10 +371,10 @@ endif()
 
 add_lib_info(${CMAKE_CURRENT_LIST_DIR}/src/support/libinfo.cc)
 
-add_library(tvm_objs OBJECT ${COMPILER_SRCS} ${RUNTIME_SRCS})
```

Comment:
decoupling runtime source from`tvm_objs` might require cross check other locations that refers to it. 

e.g. https://github.com/apache/tvm/blob/main/CMakeLists.txt#L396 

So that the runtime source and compiler part build with the same flag

Reply:
These defines were already broken/inconsistent, but I've fixed them.